### PR TITLE
11.0 mig stock inventory valuation report

### PR DESCRIPTION
**Pull request by wildi1**
Backport of stock_inventory_valuation_report to version 11.0

### CODE REVIEW

**Comment by rousseldenis** [comment]
@wildi1 Could you add 'reporting-engine' to oca_dependencies.txt ?

File: stock_inventory_valuation_report/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+* Pimolnat Suntian <pimolnats@ecosoft.co.th>
```
Comment (ps-tubtim) [inline stock_inventory_valuation_report/readme/CONTRIBUTORS.rst:1]: Please add you as contributor. :)

Comment (ps-tubtim) [reply stock_inventory_valuation_report/readme/CONTRIBUTORS.rst:1]: @wildi1

Comment (wildi1) [reply stock_inventory_valuation_report/readme/CONTRIBUTORS.rst:1]: @ps-tubtim 
Job done ;-)

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by rousseldenis** [comment]
@wildi1 Could you squash your commits before merge ?

**Comment by wildi1** [comment]
@rousseldenis 
I have squashed the commits. So you can merge just the last commit.

**Review by ps-tubtim — APPROVED** [review]

**Comment by wildi1** [comment]
Why not, or when will that be merged ?

**Comment by rousseldenis** [comment]
@wildi1 Do you want to let the two commits as is ('[ADD] Backport of stock_inventory_valuation_report' and 'Add OCA dependencies and author') before merge ?

**Comment by wildi1** [comment]
@rousseldenis Please megre it as it is. It has also been labeled with ready to merge before.

**Comment by ps-tubtim** [comment]
@wildi1 Please squash migration commits into one before merge.

**Pull request merged**